components/brave_wallet/browser/keyring_service_unittest.cc

```diff
@@ -3983,7 +3983,15 @@ TEST_F(KeyringServiceUnitTest, SignCip30MessageByCardanoKeyring) {
                 message));
 }
 
-TEST_F(KeyringServiceUnitTest, SignCip30MessageByCardanoKeyring_TestVectors) {
+#if BUILDFLAG(IS_ANDROID)
```

Comment:
not sure what is going on with this one, it wasn't fixed by adding the missing data file cc @brave/crypto-wallets-core for follow-up

Reply:
probably create an issue so we don't forget to follow up

Reply:
https://github.com/brave/brave-browser/issues/48607